Source/Core/DolphinQt2/Config/Graphics/GraphicsChoice.cpp

```diff
@@ -14,12 +15,14 @@ GraphicsChoice::GraphicsChoice(const QStringList& options, const Config::ConfigI
           &GraphicsChoice::Update);
   setCurrentIndex(Config::Get(m_setting));
 
-  if (Config::GetActiveLayerForConfig(m_setting) != Config::LayerType::Base)
-  {
-    QFont bf = font();
-    bf.setBold(true);
-    setFont(bf);
-  }
+  connect(&Settings::Instance(), &Settings::EmulationStateChanged, [this]() {
+    if (Config::GetActiveLayerForConfig(m_setting) != Config::LayerType::Base)
```

Comment:
Won't this bold settings, but never unbold them? Unlike the other conditional in GraphicsSlider, or is this intentional?

Reply:
Oh, oops. yeah this will never unbold that lol